pkg/controller/migmigration/quiesce.go

```diff
@@ -0,0 +1,78 @@
+package migmigration
+
+import (
+	"context"
+
+	appsv1 "github.com/openshift/api/apps/v1"
+	coreappsv1 "k8s.io/api/apps/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// All quiesce functionality should be put here
```

Comment:
TODO: We should add ability to quiesce other things like DaemonSets, StatefulSets, and ReplicaSets. To do this we will need to do a hacky workaround like selecting a bunk nodeSelector.

Reply:
This workaround worked well when I tried it, the strange part about it is likely that folks won't expect to have a new, invalid nodeselector set on their resources on the source cluster. It's not something you'd ever do in normal operation (unlike scaling down a deployment to 0, for example)